File: DreamsDontWorkUnlessYouDo/hungry/prob1.1/prob1.1/FizzBuzz.cpp
#include "stdafx.h"
#include <stdio.h>
#include <stdlib.h>
#include "conio.h"
int main(){
int noOfTests,*eachTests,i,j;
scanf("%d",&noOfTests);
eachTests = (int *)malloc((noOfTests+1)*sizeof(int));
for(i=0;i<noOfTests;i++){
	scanf("%d",&eachTests[i]);
}
for(i=0;i<noOfTests;i++){
	for(j=0;j<eachTests[i];j++){
		if(j%3==0 && j%5==0){
			printf("FizzBuzz\n");
		}else if(j%3==0){
			printf("Fizz\n");
		}else if(j%5==0){
			printf("Buzz\n");
		}else{
			printf("%d\n",j);
		}
	}
}
getch();
return 0;
}

File: LeetCode/fizz-buzz.java
public class Solution {
    public List<String> fizzBuzz(int n) {
       int i=0;
       List<String> res = new ArrayList<String>();
       while(i<n){
		   if((i+1)%3==0 && (i+1)%5==0){
			   res.add("FizzBuzz");
		   }else if((i+1)%3 ==0){
			   res.add("Fizz");
		   }else if((i+1)%5 ==0){
			   res.add("Buzz");
		   }else{
			   res.add(""+(i+1));
		   }
		   i++;
       }
       return res;	   
    }
}